logging/cluster-logging-loki.adoc

```diff
@@ -1,23 +1,30 @@
 :_content-type: ASSEMBLY
 :context: cluster-logging-loki
 [id="cluster-logging-loki"]
-= Loki
+= Logging using LokiStack
 include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-include::modules/cluster-logging-loki-about.adoc[leveloffset=+1]
+In {logging} documentation, _LokiStack_ refers to the {logging} supported combination of Loki, and web proxy with {product-title} authentication integration. LokiStack's proxy uses {product-title} authentication to enforce multi-tenancy. _Loki_ refers to the log store as either the individual component or an external store.
+
+Loki is a horizontally scalable, highly available, multi-tenant log aggregation system currently offered as an alternative to Elasticsearch as a log store for the {logging}. Elasticsearch indexes incoming log records completely during ingestion. Loki only indexes a few fixed labels during ingestion, and defers more complex parsing until after the logs have been stored. This means Loki can collect logs more quickly. As with Elasticsearch, you can query Loki link:https://grafana.com/docs/loki/latest/[using JSON paths or regular expressions].
```

Comment:
Probably because I didn't write this content, it was preexisting from Libby (who is also working on the other PR 🙂) - thanks for pointing this out, this is a good use case for adding a snippet maybe in the future; I'll make a note of it to reduce duplication / maintenance.